### PR TITLE
Create the light object page.

### PR DESCRIPTION
**Pull request by Bouh**
The created page: https://gdevelop-wiki-git-fork-bouh-add-light-page-with-23701e-gdevelop.vercel.app/gdevelop5/objects/light/

I used caption for images: https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions

- [x] Improve the page with illustration of the light texture.
- [x] Improve the lights example with an object using the light texture.

### CODE REVIEW

File: docs/gdevelop5/all-features/index.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Note that GDevelop can also be extended with extensions: take a look at [the lis
 * [File system](https://wiki.gdevelop.io/gdevelop5/all-features/filesystem) ([reference](https://wiki.gdevelop.io/gdevelop5/all-features/filesystem/reference))
 * [Firebase](https://wiki.gdevelop.io/gdevelop5/all-features/firebase) ([reference](https://wiki.gdevelop.io/gdevelop5/all-features/firebase/reference))
 * [Leaderboards](https://wiki.gdevelop.io/gdevelop5/all-features/leaderboards) ([reference](https://wiki.gdevelop.io/gdevelop5/all-features/leaderboards/reference))
-* [Lights](https://wiki.gdevelop.io/gdevelop5/all-features/lighting/reference)
+* [Lights](https://wiki.gdevelop.io/gdevelop5/objects/light) ([reference](https://wiki.gdevelop.io/gdevelop5/all-features/lighting/reference))
```
Comment (D8H) [inline docs/gdevelop5/all-features/index.md:57]: Relative links should be preferred.

File: docs/gdevelop5/objects/light.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+By default, the light object's rays are cast around the light object and interact with the edges of other objects using the Light Obstacle behaviour.
+
+When the collision mask of an object using the Light Obstacle behaviour is changed, the light ray will support the collision mask.
+
+### Using the light texture
+
+The light texture property allow to create a custom shape for the light.
+
+An example of a texture that illuminates in a cone shape, like it can be done for a flashlight:
```
Comment (D8H) [inline docs/gdevelop5/objects/light.md:19]:
```suggestion
For instance, a flashlight effect can be achieved by using a cone-shaped texture.
```

File: docs/gdevelop5/objects/light.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+<figure markdown>
+  ![The result of a light texture in game](result_light_texture.png)
+  <figcaption>The result in game.</figcaption>
+</figure>
+
+
+
+!!! tip
+
+    For example a light texture can be used to make a headlight bulbs for a car or a flashlight that illuminate only in one direction.
```
Comment (D8H) [inline docs/gdevelop5/objects/light.md:35]:
```suggestion
    For example, a light texture can be used to make a headlight bulbs for a car or a flashlight that illuminate only in one direction.

```

File: docs/gdevelop5/objects/light.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    Open the example below to see how custom shape for light are used.
```
Comment (D8H) [inline docs/gdevelop5/objects/light.md:36]:
```suggestion
    Open the following example to see how light custom shapes can be used.

```

File: docs/gdevelop5/objects/light.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    
+
+
+## Examples
+
+[Open example in GDevelop](https://editor.gdevelop.io/?project=example://lights){ .md-button .md-button--primary }
```
Comment (D8H) [inline docs/gdevelop5/objects/light.md:42]: It feels strange to have a title between the button and the sentence that introduces it.

File: docs/gdevelop5/objects/light.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+# Light
+
+A light object is an object that simulates a light source in 2D and cast shadow using other objects.
+The light object has several properties that can be customized, such as the color, the radius of the light, and a texture to give a shape to the light.
+
+## How does it work?
+
+By default, the light object's rays are cast around the light object and interact with the edges of other objects using the Light Obstacle behaviour.
+
+When the collision mask of an object using the Light Obstacle behaviour is changed, the light ray will support the collision mask.
```
Comment (Bouh) [inline docs/gdevelop5/objects/light.md:13]:
```suggestion
Light rays will take into account the custom collisions masks on the objects with the Light Obstacle behavior.
```

**Pull request merged**